### PR TITLE
Improve texts, fix demo link, and fix typos

### PR DESCRIPTION
**Pull request by cadamini**
@dnoneill Interesting project. The Ruby plugin is great. I plan to use it so I have read some code and the docs. 

In this pull request, I try to improve the docs. There is more work to be done in `demo/docs/quickstart/setup.md` but this is a start.

I'll be happy to help.

### CODE REVIEW

File: demo/docs/quickstart/searchform.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ title: Create a Search Form
 parent: Quickstart
 nav_order: 3
 ---
-The only really important thing when creating the search form is to make sure the "name" field matches the "searchfield" in the fields in _config.yml file you would like to search against.
 
-An example of this form can be seen below. This is the code being used in the [demo]({{site.baseurl}}/demo)
+When creating the search form, make sure the "name" field matches the "searchfield" in the fields in _config.yml file you would like to search against.
+
+This search form example code is being used in the [demo]({{site.baseurl}}/demo):
```
Comment (cadamini) [inline demo/docs/quickstart/searchform.md:10]: does it require a real URL?

**Comment by dnoneill** [comment]
@cadamini Thank you for these updates. I am happy to accept any help you are willing to give.

**Comment by cadamini** [comment]
Unfortunately, my changes have never been updated on your demo site :/

**Comment by dnoneill** [comment]
@cadamini Sorry I forgot this repo required a rebuild. It should be done now.

**Pull request merged**